# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/parent/FileSystemManagerParent.cpp
# Commit: b3fc2ce9d8cf
# Full Hash: b3fc2ce9d8cfd1ad2f7ca48d484cf060dcee0bc9
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2023-02-07 16:42:31
# Regressor Bug: 1809064
# File Overlap Count: 1
# Description:
#   Bug 1809064 - Expose FileSystemDataManager::mIOTarget; r=dom-storage-reviewers,jari
#   
#   This patch also fixes the misnamed getter for mIOTaskQueue.
#   
#   Depends on D166344
# ==============================================================================

diff -r e0d6b8852f52 -r b3fc2ce9d8cf dom/fs/parent/FileSystemManagerParent.cpp
--- a/dom/fs/parent/FileSystemManagerParent.cpp	Tue Feb 07 08:27:06 2023 +0000
+++ b/dom/fs/parent/FileSystemManagerParent.cpp	Tue Feb 07 08:27:07 2023 +0000
@@ -465,11 +465,11 @@
 
   mRequestedAllowToClose.Flip();
 
-  InvokeAsync(mDataManager->MutableIOTargetPtr(), __func__,
+  InvokeAsync(mDataManager->MutableIOTaskQueuePtr(), __func__,
               [self = RefPtr<FileSystemManagerParent>(this)]() {
                 return self->SendCloseAll();
               })
-      ->Then(mDataManager->MutableIOTargetPtr(), __func__,
+      ->Then(mDataManager->MutableIOTaskQueuePtr(), __func__,
              [self = RefPtr<FileSystemManagerParent>(this)](
                  const CloseAllPromise::ResolveOrRejectValue& aValue) {
                self->Close();